src/js/components/table.js

```diff
@@ -8,10 +8,20 @@ const SORTED = "aria-sort";
 const ASCENDING = "ascending";
 const DESCENDING = "descending";
 const SORT_OVERRIDE = "data-sort-value";
-const ICON_PATH = "../../dist/img/sprite.svg";
-const ICON_UNSORTED = `${ICON_PATH}#swap_vert`;
-const ICON_ASCENDING = `${ICON_PATH}#arrow_upward`;
-const ICON_DESCENDING = `${ICON_PATH}#arrow_downward`;
+
+const ICON_SOURCE = `
+  <svg class="usa-icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <g id="descending">
+      <path d="M19,15l-1.41-1.41L13,18.17V2H11v16.17l-4.59-4.59L5,15l7,7L19,15z"/>
+    </g>
+    <g id="ascending">
+      <path transform="rotate(180, 12, 12)" d="M19,15l-1.41-1.41L13,18.17V2H11v16.17l-4.59-4.59L5,15l7,7L19,15z"/>
+    </g>
+    <g id="unsorted" >
+      <polygon points="15.17 15 13 17.17 13 6.83 15.17 9 16.58 7.59 12 3 7.41 7.59 8.83 9 11 6.83 11 17.17 8.83 15 7.42 16.41 12 21 16.59 16.41 15.17 15"/>
+    </g>
+  </svg>
+`;
```

Comment:
@jaredcunha I've included the src for all three icons in this svg, and I use some CSS to control the fill color based on which group/node within the SVG should be active

Reply:
I like this approach!